modules/cnv_using_openvswitch_bridge.adoc

```diff
@@ -70,15 +69,15 @@ spec:
     }'
 ....
 +
-NOTE: `"vlan"` is optional. If omitted, the VMI will be attached
```

Comment:
This should be in a note box per our[ guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions) on admonitions.